zilencer/management/commands/populate_db.py

```diff
@@ -386,6 +386,7 @@ def handle(self, **options: Any) -> None:
                 "Scotland": {"description": "Located in the United Kingdom"},
                 "Venice": {"description": "A northeastern Italian city"},
                 "Rome": {"description": "Yet another Italian city", "is_web_public": True},
+                "Hamm": {"description": "A city in Germany", "is_web_public": True},
```

Comment:
Maybe pick London?  I think `Hamm` might look like a typo to folks who don't know it's a city.